ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -70,15 +70,7 @@ export class DkgCreateCommand extends IronfishCommand {
 
     if (flags.ledger) {
       ledger = new LedgerMultiSigner(this.logger)
-      try {
-        await ledger.connect()
-      } catch (e) {
-        if (e instanceof Error) {
-          this.error(e.message)
-        } else {
-          throw e
-        }
-      }
```

Comment:
This happens in a lot of places, needs to be replaced everywhere. Can happen later though